ring/example/local/go.mod

```diff
@@ -0,0 +1,74 @@
+module github.com/grafana/dskit/ring/example/local
+
+go 1.19
+
+require (
+	github.com/go-kit/log v0.2.1
+	github.com/grafana/dskit v0.0.0-00010101000000-000000000000
```

Comment:
Can we use the latest main revision?

Reply:
The examples in `ring/example/local` always uses it's local version of dskit anyway.
https://github.com/grafana/dskit/blob/kavirajk%2Fring-examples/ring/example/local/go.mod#L74

Reply:
Aha, I missed the `replace` directive somehow.